contentcuration/contentcuration/frontend/channelEdit/views/trash/__tests__/trashModal.spec.js

```diff
@@ -0,0 +1,159 @@
+import { mount } from '@vue/test-utils';
+import VueRouter from 'vue-router';
+import TrashModal from '../TrashModal';
+import store from '../../../store';
+import { RouterNames } from '../../../constants';
+
+const testChildren = [
+  {
+    id: 'test1',
+    title: 'Item',
+    kind: 'video',
+    modified: new Date(2020, 1, 20),
+  },
+  {
+    id: 'test2',
+    title: 'Item',
+    kind: 'audio',
+    modified: new Date(2020, 2, 1),
+  },
+  {
+    id: 'test3',
+    title: 'Topic',
+    kind: 'topic',
+    modified: new Date(2020, 1, 1),
+  },
+];
+
+const router = new VueRouter({
+  routes: [
+    {
+      name: 'PARENT_ROUTE',
+      path: '/',
+      children: [
+        {
+          name: RouterNames.TRASH,
+          path: 'trash',
+          component: TrashModal,
+          props: true,
+        },
+      ],
+    },
+  ],
+});
+
+function makeWrapper(items) {
+  return mount(TrashModal, {
+    store,
+    router,
+    attachToDocument: true,
```

Comment:
Why is there a need to set `attachToDocument` to `true`? Is it because of event listeners in `ResizableNavigationDrawer` or is there another reason? I wanted to suggest to rather mock such components though I can see that the whole `ResourceDrawer` is stubbed so not 100% sure.